extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -833,6 +833,12 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
       Xss.sanitizePrepend(el, '<img src="/img/svgs/expired-timer.svg" class="expired-time">');
       $(el).attr('title', 'Does use encryption but their public key is expired. You should ask them to send ' +
         'you an updated public key.' + this.recipientKeyIdText(contact));
+    } else if (contact.revoked) {
+      recipient.status = RecipientStatus.REVOKED;
+      $(el).addClass("revoked");
+      Xss.sanitizePrepend(el, '<img src="/img/svgs/expired-timer.svg" class="expired-time">');
```

Comment:
Should we have some other image here for revoked key?

Reply:
@limonte could you add a similar icon into svgs that has maybe a stop sign? Or exclamation mark?

Reply:
I've made a separate issue - we can fix the icon separately afterwards. Thanks for noticing.

Reply:
The icon has been added in #3662